hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/HoodieTable.java

```diff
@@ -316,6 +317,13 @@ public HoodieTimeline getCleanTimeline() {
     return getActiveTimeline().getCleanerTimeline();
   }
 
+  /**
+   * Get rollback timeline.
+   */
+  public HoodieTimeline getRollbackTimeline() {
```

Comment:
we should avoid these sort of helpers unless one exists for each of the other actions. if not, lets remove